vlsv_writer.cpp

```diff
@@ -680,4 +715,83 @@ namespace vlsv {
       return checkSuccess(success,comm);
    }
 
+
```

Comment:
It looks like this violates dry run; if dryRunning is true, you must perform all the actions that you'd normally do but do not perform actual writes. Put data to buffer and clear it when emptyBuffer is called, but do not call MPI writes etc.